include/ofi.h

```diff
@@ -63,6 +63,7 @@
 extern "C" {
 #endif
 
+#define OFI_SKIP_LOOPBACK	(1ULL << 58)
```

Comment:
We should be able to use FI_REMOTE_COMM to filter out loopback devices.  That said, I'm not sure that we would always want to do this.  Being able to test multi-rail on a single system using multiple endpoints would be useful.

Reply:
Doesn't FI_REMOTE_COMM just request providers capable of doing remote communication and not actually filter loopback devices?

On second thoughts, we may not need this filtering if we make the provider read the rails that have to be used through env variable or hints. That should work for loopback devices as well if it's needed.